Update Select files and settings to sync.md (#368)
obsidianmd/obsidian-docs: en/Obsidian Sync/Select files and settings to sync.md

**Notes:**

- Synced files remain in your remote vault even if you exclude them later on. If possible, configure the files and settings you want to sync before you start syncing your vault.

> - If you sync any vault settings, changes will only be updated during start-up. After changing a setting on one device, you will need to restart Obsidian on your other devices for the new changes to take effect. (For example, if you change your Daily Notes' path in the Daily Notes plugin, then create a new daily note on another device without restarting Obsidian, it will use your previous path.)

Reviewer: ```suggestion
- Obsidian only applies vault settings during start-up. If you change a setting on one device, you need to restart Obsidian on your other devices for the new changes to take effect. For example, if you change the path of your daily notes in the [[Daily Notes]] plugin, you need to restart Obsidian on your other devices to use the new path.
```

Rewriting it like this, I'm not sure if the example is needed. WDYT?

Author: I think the example is helpful, otherwise it's kind of abstract. Otherwise it looks good!

> - ==Sync doesn't sync Sync's settings.== This allows users to configure Sync differently on each device according to their needs. This means, however, that you must configure custom Sync settings on each device.

Reviewer: ```suggestion
- To allow device-specific configuration, Obsidian Sync doesn't sync its own settings. You need to configure Sync for each of your devices.
```
